docs/puml/HelpAddDiet.puml

```diff
@@ -0,0 +1,29 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+skinparam Style strictuml
+skinparam SequenceMessageAlignment center
+participant ":AthletiCLI" as AthletiCLI #lightblue
+participant "<<class>>\nParser" as Parser #lightgreen
+participant ":HelpCommand" as HelpCommand #lightpink
+participant ":Ui" as Ui #lightyellow
+actor User as User
+
+
+'autonumber
+AthletiCLI++
+AthletiCLI -> Ui++ : getUserCommand()
+User -> Ui : "help add-diet"
+Ui --> AthletiCLI-- : "help add-diet"
+AthletiCLI -> Parser++ : parseCommand("help add-diet")
+create HelpCommand
+Parser -> HelpCommand++ : new HelpCommand("add-diet")
+HelpCommand --> Parser-- : :HelpCommand
+Parser --> AthletiCLI-- : :HelpCommand
+AthletiCLI -> HelpCommand++ : execute(data)
+HelpCommand --> AthletiCLI-- : feedback:String
+AthletiCLI -> Ui++ : showMessages(feedback)
+Ui --> AthletiCLI--
+
+destroy HelpCommand
```

Comment:
Overall I think this is a pretty good sequence diagram, however, I think it might be better to separate the cross sign and the dash line of your method, you may see the marks in the following screenshot
![IMG_1481](https://github.com/nus-cs2113-AY2324S1/tp/assets/139958049/0a22141c-c948-4ff7-b122-2c35039e4928)